bedrock/firefox/templates/firefox/developer.html

```diff
@@ -223,9 +223,7 @@
     </div>
     {# See Bug 1095176 #}
```

Comment:
Should we update the bug number for reference here? (or remove if it's no longer really warranted).